packages/shared/src/components/onboarding/InteractiveFeedStep.tsx

```diff
@@ -0,0 +1,134 @@
+import type { ReactElement } from 'react';
+import React, { useState, useContext } from 'react';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../typography/Typography';
+import { TagSelection } from '../tags/TagSelection';
+import { SearchField } from '../fields/SearchField';
+import { useTagSearch } from '../../hooks/useTagSearch';
+import { Origin } from '../../lib/log';
+import useDebounceFn from '../../hooks/useDebounceFn';
+import Feed from '../Feed';
+import { OtherFeedPage, RequestKey } from '../../lib/query';
+import { PREVIEW_FEED_QUERY } from '../../graphql/feed';
+import { FeedLayoutProvider } from '../../contexts/FeedContext';
+import AuthContext from '../../contexts/AuthContext';
+import useFeedSettings from '../../hooks/useFeedSettings';
+import { REQUIRED_TAGS_THRESHOLD } from './common';
+import { useInteractiveFeedContext } from '../../contexts/InteractiveFeedContext';
+import ProgressCircle from '../ProgressCircle';
+import { interactiveFeedEmpty } from '../../lib/image';
+import { Image } from '../image/Image';
+
+const InteractiveFeedStep = (): ReactElement => {
+  const { user } = useContext(AuthContext);
+  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [onSearch] = useDebounceFn(setSearchQuery, 200);
+  const { feedSettings } = useFeedSettings({});
+  const { completion } = useInteractiveFeedContext();
+  const { data: searchResult } = useTagSearch({
+    value: searchQuery,
+    origin: Origin.EditTag,
+  });
+  const searchTags = searchResult?.searchTags.tags || [];
+  const tagsCount = feedSettings?.includeTags?.length || 0;
+  const hasMinimumTags = tagsCount >= REQUIRED_TAGS_THRESHOLD;
+
+  return (
+    <div className="flex w-full max-w-[85rem] flex-row gap-6">
+      <section className="flex w-1/4 flex-col gap-6">
```

Comment:
This doesn't scale mobile now, not sure if it's intented or not?

Reply:
Yeah, at the moment we are only testing it for Laptop sizes 🙏

Reply:
Where does it force laptop only then?

Reply:
cc @AmarTrebinjac can you validate that we only do the enrollment only when it is Laptop?